cli/ballerina-tool/src/main/resources/cli-help/ballerina-test.help

```diff
@@ -53,6 +53,14 @@ OPTIONS
             Specifies the test functions to be executed. A specific set of
             test functions can be executed using this command.
 
+       --tests <PackageName>:<test_function_name, ...>
+            Specifies the test functions to be executed from the default
+            module only. Wild cards can be used to execute specific groups of tests.
+
+       --tests <PackageName>.<ModuleName>:<test_function_name, ...>
+            Specifies the test functions to be executed from the module
+            specified. Wild cards can be used to execute specific groups of tests.
+
```

Comment:
shall we add how to execute all tests of a particular module as the sample? Because it's the most common use case.
Then, let's add that other wildcards can be used.

Reply:
+1, lets add the flag explanation in a general and move the examples